main.css

```diff
@@ -1 +1,75 @@
-h1 { color: hsl(0, 82%, 57%); }
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: 'inter', sans-serif;
+}
+
+.header {
+  max-width: 375px;
+  display: flex;
+  flex-direction: column;
+  margin: 0 auto;
+}
+
+.header .navSec {
+  width: 100%;
+  height: 75px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.header .banner {
+  width: 100%;
+  height: 650px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  align-items: center;
+  background-image: url("/images/header-illsutration-mobile.svg");
```

Comment:
- Good job using the `background-image` property to place the background image section, but when navigating into the images dir please place a `.` before the `/`. Please fix this for this image and all other instances where you had to navigate into the images dir. 👍 👍